spec/waypoint_spec.js

```diff
@@ -530,23 +535,28 @@ describe('<Waypoint>', function() {
 
       // now show it and we should get an onEnter
       node.style.position = 'relative';
-      scrollNodeTo(this.component, 10);
+      scrollNodeTo(this.component, 100);
       expect(this.props.onEnter).
         toHaveBeenCalledWith({
           currentPosition: Waypoint.inside,
-          previousPosition: Waypoint.invisible,
+          previousPosition: Waypoint.below,
```

Comment:
The `invisible`ness was sort of a workaround due to limitations in the way we were calculating distance to the top. I think this might actually be better. Would you suggest removing that "position" after your change?